docs/src/add-services/influxdb.md

```diff
@@ -47,17 +47,24 @@ if (getenv('PLATFORM_RELATIONSHIPS')) {
 
 ## Exporting data
 
-InfluxDB includes its own [export mechanism](https://docs.influxdata.com/influxdb/v1.2/tools/influx_inspect/).
-To gain access to the server from your local machine open an SSH tunnel with the Platform.sh CLI:
+To export your data from InfluxDB, follow these steps:
 
-```bash
-platform tunnel:single --relationship {{< variable "RELATIONSHIP_NAME" >}}
-```
+1. Install and set up the [`influx` CLI](https://docs.influxdata.com/influxdb/cloud/tools/influx-cli/).
+2. Connect to your InfluxDB service with the [Platform.sh CLI](../administration/cli/_index.md):
 
-By default, this opens a tunnel at `127.0.0.1:30000`.
+   ```bash
+   platform tunnel:open --host {{<variable "RELATIONSHIP_NAME" >}}
```

Comment:
:hammer_and_wrench: `host` is a deprecated flag. We shouldn't be recommending it. I would still recommend using `tunnel:single` as more focused on the task at hand. (Plus not requiring extra PHP extensions.)

```suggestion
   platform tunnel:single --relationship {{<variable "RELATIONSHIP_NAME" >}}
```

Reply:
Or if you don't want to put the flag in, it's fine to let them choose through the CLI:
```suggestion
   platform tunnel:single
```